tcms/testruns/models.py

```diff
@@ -251,14 +251,13 @@ def stats_caseruns_status(self, case_run_statuses=None):
                                          for status in case_run_statuses)
 
         for row in rows:
-            status_pk = row['case_run_status']
-            caserun_statuses_subtotal[status_pk][0] = row['status_count']
+            caserun_statuses_subtotal[row['case_run_status']][0] = row['status_count']
 
         complete_count = 0
         failure_count = 0
         caseruns_total_count = 0
 
-        for status_pk, total_info in caserun_statuses_subtotal.items():
+        for total_info in caserun_statuses_subtotal.items():
```

Comment:
use `_status_pk` for the unused variable. `.items()` returns 2 elements, not one, that's why your test fails

Reply:
should it be ignored then?

Reply:
No. The pylint error is genuine because we don't use the status_pk variable. It is there in the expression because .items() returns tuple and you want total_info to be the value, where status_pk is the key. Prefixing the variable name with `_` (instead of replacing with _) achieves 2 things:
1) tells pylint to ignore the fact that the variable is unused
2) keeps the rest of the variable name so that we know what it is when we read the code.